TurfRegressionTests/Common/UnindexedTreesCollection.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+import Turf
+
+final class UnindexedTreesCollection: Collection {
```

Comment:
Ha, good question - I was hoping to test somehow that we don't even have the same methods, or that accessing the `indexed` property would either be `nil` or would throw an error.

I then kinda realised that Swift probably means I won't be able to (I was thinking along the lines of calling 'performSelector' or similar, no dice)

You think its OK to just have an indexed collection and call some of the Query related methods?